src/api/app/controllers/webui/package_controller.rb

```diff
@@ -46,8 +46,6 @@ class Webui::PackageController < Webui::WebuiController
                                             :update_build_log, :devel_project, :buildresult, :rpmlint_result,
                                             :rpmlint_log, :meta, :attributes, :repositories, :files]
 
-  before_filter :do_backend_login, only: [:save_meta, :abort_build, :trigger_rebuild, :wipe_binaries, :remove]
```

Comment:
I removed ``:remove`` and ``:save_meta`` because they also communicate now directly with the backend!
(@hennevogel you refactored the remove action, is my assumption right? Just looked at it briefly)